### DISCUSSION TITLE
new learner

### DISCUSSION THREAD

**Discussion by Qata-Mo7ammad-Isma3el** [mrdbourke/pytorch-deep-learning#1101] [Q&A]
**can someone please  suggest some  real projects to apply what iv learned from the first four sections (introduction , fundamentals, workflow , neural networks classification)**

**Comment by iashyam** [comment]
Hey
This might help you
https://github.com/iashyam/Machine-Learning-Projects/wiki/40-Projects-Roadmap-to-Deep-Learning
